kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/kview.css

```diff
@@ -4092,3 +4092,117 @@ Styling a context menu for kview ui/ux controls
     -fx-background-insets: 0;
     -fx-background-radius: 6;
 }
+
+/*****************************
+* next gen reasoner styles
+*****************************/
+.reasoner-result-container .label {
+    -fx-text-fill: -Grey-1;
+    -fx-font-family: "Noto Sans";
+    -fx-font-size: 14px;    
+}
+
+.reasoner-result-container .close {
+    -height: 18px;
+    -width: 18px;
+    -fx-min-height: -height;
+    -fx-pref-height: -height;
+    -fx-max-height: -height;
+    -fx-content-display: graphic-only;
+    -fx-min-width: -width;
+    -fx-pref-width: -width;
+    -fx-max-width: -width;
+    -fx-background-color: -Grey-1;
+    -fx-shape: "M19 6.41L17.59 5L12 10.59L6.41 5L5 6.41L10.59 12L5 17.59L6.41 19L12 13.41L17.59 19L19 17.59L13.41 12L19 6.41Z";
+}
+
+.titled-pane.classification-date {
+    -fx-text-fill: white; 
+    -fx-border-width: 1 0 0 0;
+    -fx-border-color: -Grey-9;    
+}
+
+
+.titled-pane.classification-date > .title {        
+    -fx-background-color: -Grey-11;
+    -fx-pref-height: 32;
+    -fx-padding: 8 4 4 4;
+}
+
+.titled-pane.classification-date > .title > .arrow-button .arrow {
+    -width-size: 15px;
+    -height-size: 9px;
+    -fx-min-width: -width-size;
+    -fx-pref-width: -width-size;
+    -fx-max-width: -width-size;
+
+    -fx-min-height: -height-size;
+    -fx-pref-height: -height-size;
+    -fx-max-height: -height-size;
+    -fx-shape: "M7.41 8.29508L6 9.70508L12 15.7051L18 9.70508L16.59 8.29508L12 12.8751L7.41 8.29508Z";
+    -fx-background-color: -Secondary-05;    
+}
+
+.reasoner-entry-titled-pane {
+    -fx-text-fill: -Grey-1; 
+    -fx-font-size: 13px;        
+    -fx-font-weight: bold;    
+}
+
+
+.reasoner-entry-titled-pane > .title {        
+    -fx-background-color: -Grey-8;        
+}
+
+.reasoner-entry-titled-pane .content {
+    -fx-background-color: -Grey-8;
+    -fx-border-width: 0 0 0 0;
+}
+
+.reasoner-label-container {
+    -fx-background-color: white;        
+}
+
+.titled-pane.reasoner-entry-titled-pane:focused > .title .arrow-button .arrow {
```

Comment:
Can we shorten the name of the style class `.reasoner-entry-titled-pane` to `.reasoner-entry`? 

```css
.titled-pane.reasoner-entry:focused > .title .arrow-button .arrow { 
...
}
```

Reply:
fixed